velox/functions/sparksql/tests/DateTimeFunctionsTest.cpp

```diff
@@ -757,5 +757,70 @@ TEST_F(DateTimeFunctionsTest, fromUnixtime) {
       fromUnixTime(0, "yyyy-MM-dd HH:II"), "Specifier I is not supported");
 }
 
+TEST_F(DateTimeFunctionsTest, makeYMInterval) {
+  const auto makeYMInterval = [&](const std::optional<int32_t>& year,
+                                  const std::optional<std::int32_t>& month) {
+    return evaluateOnce<int32_t, int32_t>(
+        "make_ym_interval(c0, c1)",
+        {year, month},
+        {INTEGER(), INTEGER()},
+        std::nullopt,
+        {INTERVAL_YEAR_MONTH()});
+  };
+
+  const auto makeYMIntervalOnlyYear = [&](const std::optional<int32_t>& year) {
+    return evaluateOnce<int32_t, int32_t>(
+        "make_ym_interval(c0)",
+        {year},
+        {INTEGER()},
+        std::nullopt,
+        {INTERVAL_YEAR_MONTH()});
+  };
+
+  const auto getYMInterval = [](const std::optional<int32_t>& ymInterval) {
+    return ymInterval.has_value()
+        ? std::make_optional(INTERVAL_YEAR_MONTH()->valueToString(*ymInterval))
+        : std::nullopt;
+  };
+
+  EXPECT_EQ(getYMInterval(makeYMInterval(1, 2)), "1-2");
```

Comment:
Can `makeYMInterval` return `getYMInterval` result, then the test will be simplfied.